### PR TITLE
Addition-of-mobile-menu-pull-request

### PR DESCRIPTION
**Pull request by UsamaMasood12**
In this project, I added the mobile menu of my portfolio, which only works for mobile screens with help of js code.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -26,10 +26,65 @@
...
 .menu {
+  background-color: #3c3a39;
   position: absolute;
-  top: 55px;
-  left: 330px;
+  top: 45px;
+  left: 300px;
   margin-top: -30px;
+  border: none;
+}
```
Comment (Davidosky007) [inline style.css]: * Kindly ensure that your menu drop-down fits and is responsive as it currently has a horizontal scroll which gives it not so good user experience below is a recording of the issue 

https://user-images.githubusercontent.com/52497246/208438086-8d298f91-dfef-4493-8d7f-9cc5b0f22f45.mp4

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @UsamaMasood12 @zewdu444 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Good looking readme file 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @UsamaMasood12

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] The horizontal scroll actually scrolls the popup's header up which is not very professional. As mentioned by the previous reviewer, kindly make the popup header fixed like the body

**current design**
image
 

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -535,10 +592,9 @@ ul.points {
   display: flex;
   list-style-type: none;
   gap: 15px;
-  position: relative;
   right: 35px;
   width: 190px;
```
Comment (cindykandie) [inline style.css:596]: Kindly avoid using fixed widths to have more responsiveness, try using rem/em values

**Review by cindykandie — COMMENTED** [review]
Good work

**Comment by batoorsohail** [comment]
# Highlights:
- No Linters error 👍 
- Well-written PR description 💯 
- Clean Code 🥇

**Comment by dieum512** [comment]
I really appreciate this work a lot, but it can be a lot better if you ad the live demo link to your README file to help people see the way your deployed webpage is

**Pull request merged**